src/core/ext/load_reporting/load_reporting.h

```diff
@@ -39,11 +39,11 @@
 
 /** Metadata key for initial metadata coming from clients */
 /* TODO(dgq): change to the final value TBD */
```

Comment:
remove TODO here and below? also, the description seems wrong (or too vague) (also below)